wai-saml2.cabal

```diff
@@ -20,6 +20,8 @@ build-type:     Simple
 extra-source-files:
     README.md
     CHANGELOG.md
+    tests/data/google.xml
```

Comment:
Looks like this has snuck in from the other PR. Probably not an issue if this gets rebased on top of that one, but just flagging it up for visibility.

Reply:
I think it's more like that this change was missing in #18. Sorry about this

Reply:
fixed in a separate PR #26

Reply:
Thanks!